<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VoxCadClass</class>
 <widget class="QMainWindow" name="VoxCadClass">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>751</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VoxCad</string>
  </property>
  <property name="windowIcon">
   <iconset resource="VoxCad.qrc">
    <normaloff>:/VoxCad/VoxCad.ico</normaloff>:/VoxCad/VoxCad.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="margin">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>751</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuExport">
     <property name="title">
      <string>Export</string>
     </property>
     <widget class="QMenu" name="menuAlternate_VXC">
      <property name="title">
       <string>Alternate VXC</string>
      </property>
      <addaction name="actionBASE64_VXC"/>
      <addaction name="actionASCII_VXC"/>
     </widget>
     <widget class="QMenu" name="menuMesh">
      <property name="title">
       <string>Mesh</string>
      </property>
      <addaction name="actionSTL"/>
     </widget>
     <addaction name="actionVXA_OUT"/>
     <addaction name="actionKV6"/>
     <addaction name="actionDMU_Out"/>
     <addaction name="actionVoxelCoordinates"/>
     <addaction name="menuMesh"/>
     <addaction name="menuAlternate_VXC"/>
    </widget>
    <widget class="QMenu" name="menuImport">
     <property name="title">
      <string>Import</string>
     </property>
     <addaction name="actionVXA_IN"/>
    </widget>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="menuImport"/>
    <addaction name="menuExport"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="menuStandard_Views">
     <property name="title">
      <string>Standard Views</string>
     </property>
     <addaction name="actionPerspective"/>
     <addaction name="separator"/>
     <addaction name="actionTop"/>
     <addaction name="actionBottom"/>
     <addaction name="actionLeft"/>
     <addaction name="actionRight"/>
     <addaction name="actionFront"/>
     <addaction name="actionBack"/>
    </widget>
    <widget class="QMenu" name="menuSection">
     <property name="title">
      <string>Section</string>
     </property>
     <addaction name="actionSection_View"/>
     <addaction name="actionLayer_Forward"/>
     <addaction name="actionLayer_Back"/>
    </widget>
    <addaction name="actionCenter_View"/>
    <addaction name="menuStandard_Views"/>
    <addaction name="menuSection"/>
    <addaction name="separator"/>
    <addaction name="actionShow_Triad"/>
    <addaction name="actionLarge_Mode"/>
    <addaction name="actionView_Tiled"/>
    <addaction name="actionGraphics_Enabled"/>
    <addaction name="separator"/>
    <addaction name="actionReference_View"/>
    <addaction name="actionInfo"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuAnalyze">
    <property name="title">
     <string>Analyze</string>
    </property>
    <addaction name="actionBCs"/>
    <addaction name="actionSolve"/>
    <addaction name="separator"/>
    <addaction name="actionTensile"/>
    <addaction name="actionPhysics"/>
    <addaction name="actionBenchmark"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <widget class="QMenu" name="menuDrawing">
     <property name="title">
      <string>Drawing</string>
     </property>
     <addaction name="actionPencil"/>
     <addaction name="actionRectangle"/>
     <addaction name="actionCircle"/>
     <addaction name="actionPaint_Bucket"/>
    </widget>
    <addaction name="actionEdit_Voxels"/>
    <addaction name="action3D_Brush"/>
    <addaction name="separator"/>
    <addaction name="menuDrawing"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCopy"/>
    <addaction name="actionCut"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionWorkspace"/>
    <addaction name="actionPalette"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuTools"/>
   <addaction name="menuAnalyze"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="FileToolBar">
   <property name="windowTitle">
    <string>File</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
  </widget>
  <widget class="QToolBar" name="ViewtoolBar">
   <property name="windowTitle">
    <string>View</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionCenter_View"/>
   <addaction name="actionPerspective"/>
   <addaction name="actionTop"/>
   <addaction name="actionBottom"/>
   <addaction name="actionLeft"/>
   <addaction name="actionRight"/>
   <addaction name="actionFront"/>
   <addaction name="actionBack"/>
  </widget>
  <widget class="QToolBar" name="DrawingtoolBar">
   <property name="windowTitle">
    <string>Draw</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionPencil"/>
   <addaction name="actionRectangle"/>
   <addaction name="actionCircle"/>
   <addaction name="actionPaint_Bucket"/>
  </widget>
  <widget class="QToolBar" name="DialogToolbar">
   <property name="windowTitle">
    <string>Dialogs</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionPalette"/>
   <addaction name="actionWorkspace"/>
   <addaction name="actionEdit_Voxels"/>
   <addaction name="action3D_Brush"/>
   <addaction name="actionInfo"/>
   <addaction name="actionReference_View"/>
   <addaction name="actionBCs"/>
   <addaction name="actionPhysics"/>
   <addaction name="actionTensile"/>
  </widget>
  <widget class="QToolBar" name="SectionToolBar">
   <property name="windowTitle">
    <string>Section</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSection_View"/>
   <addaction name="actionLayer_Back"/>
   <addaction name="actionLayer_Forward"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="VoxCad.qrc">
     <normaloff>:/Icons/Icons/Open.png</normaloff>:/Icons/Icons/Open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="VoxCad.qrc">
     <normaloff>:/Icons/Icons/Save.png</normaloff>:/Icons/Icons/Save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionCenter_View">
   <property name="icon">
    <iconset resource="VoxCad.qrc">
     <normaloff>:/Icons/Icons/ZoomExtents2.png</normaloff>:/Icons/Icons/ZoomExtents2.png</iconset>
   </property>
   <property name="text">
    <string>Zoom Extents</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionASCII_VXC">
   <property name="text">
    <string>Ascii</string>
   </property>
  </action>
  <action name="actionBASE64_VXC">
   <property name="text">
    <string>Base64</string>
   </property>
  </action>
  <action name="actionSolve">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="VoxCad.qrc">
     <normaloff>:/Icons/Icons/Analyze.png</normaloff>:/Icons/Icons/Analyze.png</iconset>
   </property>
   <property name="text">
    <string>Solve</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionPerspective">
   <property name="icon">
    <iconset resource="VoxCad.qrc">
     <normaloff>:/Icons/Icons/ViewPerspective.png</normaloff>:/Icons/Icons/ViewPerspective.png</iconset>
   </property>
   <property name="text">
    <string>Perspective</string>
   </property>
   <property name="shortcut">
    <string>Alt+1</string>
   </property>
  </action>
  <action name="actionTop">
   <property name="icon">
    <iconset resource="VoxCad.qrc">
     <normaloff>:/Icons/Icons/ViewTop.png</normaloff>:/Icons/Icons/ViewTop.png</iconset>
   </property>
   <property name="text">
    <string>Top</string>
   </property>
   <property name="shortcut">
    <string>Alt+2</string>
   </property>
  </action>
  <action name="actionBottom">
   <property name="icon">
    <iconset resource="VoxCad.qrc">
     <normaloff>:/Icons/Icons/ViewBottom.png</normaloff>:/Icons/Icons/ViewBottom.png</iconset>
   </property>
   <property name="text">
    <string>Bottom</string>
   </property>
   <property name="shortcut">
    <string>Alt+3</string>
   </property>
  </action>
  <action name="actionLeft">
   <property name="icon">
    <iconset resource="VoxCad.qrc">
     <normaloff>:/Icons/Icons/ViewLeft.png</normaloff>:/Icons/Icons/ViewLeft.png</iconset>
   </property>
   <property name="text">
    <string>Left</string>
   </property>
   <property name="shortcut">
    <string>Alt+4</string>
   </property>
  </action>
  <action name="actionRight">
   <property name="icon">
    <iconset resource="VoxCad.qrc">
     <normaloff>:/Icons/Icons/ViewRight.png</normaloff>:/Icons/Icons/ViewRight.png</iconset>
   </property>
   <property name="text">
    <string>Right</string>
   </property>
   <property name="shortcut">
    <string>Alt+5</string>
   </property>
  </action>
  <action name="actionFront">
   <property name="icon">
    <iconset resource="VoxCad.qrc">
     <normaloff>:/Icons/Icons/ViewFront.png</normaloff>:/Icons/Icons/ViewFront.png</iconset>
   </property>
   <property name="text">
    <string>Front</string>
   </property>
   <property name="shortcut">
    <string>Alt+6</string>
   </property>
  </action>
  <action name="actionBack">
   <property name="icon">
    <iconset resource="VoxCad.qrc">
     <normaloff>:/Icons/Icons/ViewBack.png</normaloff>:/Icons/Icons/ViewBack.png</iconset>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
   <property name="shortcut">
    <string>Alt+7</string>
   </property>
  </action>
  <action name="actionEdit_Voxels">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="VoxCad.qrc">
     <normaloff>:/Icons/Icons/EditLayer.png</normaloff>:/Icons/Icons/EditLayer.png</iconset>
   </property>
   <property name="text">
    <string>Edit Voxels</string>
   </property>
   <property name="shortcut">
    <string>E</string>
   </property>
  </action>
  <action name="actionLayer_Back">
   <property name="icon">
    <iconset resource="VoxCad.qrc">
     <normaloff>:/Icons/Icons/LayerBack.png</normaloff>:/Icons/Icons/LayerBack.png</iconset>
   </property>
   <property name="text">
    <string>Layer Back</string>
   </property>
   <property name="shortcut">
    <string>-</string>
   </property>
  </action>
  <action name="actionLayer_Forward">
   <property name="icon">
    <iconset resource="VoxCad.qrc">
     <normaloff>:/Icons/Icons/LayerForward.png</normaloff>:/Icons/Icons/LayerForward.png</iconset>
   </property>
   <property name="text">
    <string>Layer Forward</string>
   </property>
   <property name="shortcut">
    <string>+</string>
   </property>
  </action>
  <action name="actionPencil">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="VoxCad.qrc">
     <normaloff>:/Icons/Icons/Pencil.png</normaloff>:/Icons/Icons/Pencil.png</iconset>
   </property>
   <property name="text">
    <string>Pencil</string>
   </property>
   <property name="shortcut">
    <string>Alt+Q</string>
   </property>
  </action>
  <action name="actionRectangle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="VoxCad.qrc">
     <normaloff>:/Icons/Icons/Rectangle.png</normaloff>:/Icons/Icons/Rectangle.png</iconset>
   </property>
   <property name="text">
    <string>Rectangle</string>
   </property>
   <property name="shortcut">
    <string>Alt+W</string>
   </property>
  </action>
  <action name="actionCircle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="VoxCad.qrc">
     <normaloff>:/Icons/Icons/Ellipse.png</normaloff>:/Icons/Icons/Ellipse.png</iconset>
   </property>
   <property name="text">
    <string>Circle</string>
   </property>
   <property name="shortcut">
    <string>Alt+E</string>
   </property>
  </action>
  <action name="actionReference_View">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="VoxCad.qrc">
     <normaloff>:/Icons/Icons/Aux_View.png</normaloff>:/Icons/Icons/Aux_View.png</iconset>
   </property>
   <property name="text">
    <string>Reference View</string>
   </property>
   <property name="shortcut">
    <string>R</string>
   </property>
  </action>
  <action name="actionPalette">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="VoxCad.qrc">
     <normaloff>:/Icons/Icons/Material.png</normaloff>:/Icons/Icons/Material.png</iconset>
   </property>
   <property name="text">
    <string>Palette</string>
   </property>
   <property name="shortcut">
    <string>P</string>
   </property>
  </action>
  <action name="actionWorkspace">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="VoxCad.qrc">
     <normaloff>:/Icons/Icons/Workspace.png</normaloff>:/Icons/Icons/Workspace.png</iconset>
   </property>
   <property name="text">
    <string>Workspace</string>
   </property>
   <property name="shortcut">
    <string>W</string>
   </property>
  </action>
  <action name="actionInfo">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="VoxCad.qrc">
     <normaloff>:/Icons/Icons/Info.png</normaloff>:/Icons/Icons/Info.png</iconset>
   </property>
   <property name="text">
    <string>Info</string>
   </property>
   <property name="shortcut">
    <string>I</string>
   </property>
  </action>
  <action name="actionBCs">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="VoxCad.qrc">
     <normaloff>:/Icons/Icons/BCs.png</normaloff>:/Icons/Icons/BCs.png</iconset>
   </property>
   <property name="text">
    <string>Boundary Conditions</string>
   </property>
   <property name="shortcut">
    <string>B</string>
   </property>
  </action>
  <action name="actionSection_View">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="VoxCad.qrc">
     <normaloff>:/Icons/Icons/Section.png</normaloff>:/Icons/Icons/Section.png</iconset>
   </property>
   <property name="text">
    <string>Section View</string>
   </property>
   <property name="shortcut">
    <string>Alt+S</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="VoxCad.qrc">
     <normaloff>:/Icons/Icons/New.png</normaloff>:/Icons/Icons/New.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionVXA_OUT">
   <property name="text">
    <string>Simulation</string>
   </property>
  </action>
  <action name="actionSTL">
   <property name="text">
    <string>STL</string>
   </property>
  </action>
  <action name="actionDMU_Out">
   <property name="text">
    <string>DMU</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="VoxCad.qrc">
     <normaloff>:/Icons/Icons/Copy.png</normaloff>:/Icons/Icons/Copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="VoxCad.qrc">
     <normaloff>:/Icons/Icons/Cut.png</normaloff>:/Icons/Icons/Cut.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="VoxCad.qrc">
     <normaloff>:/Icons/Icons/Paste.png</normaloff>:/Icons/Icons/Paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionVXA_IN">
   <property name="text">
    <string>Simulation</string>
   </property>
  </action>
  <action name="actionLarge_Mode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Large Mode</string>
   </property>
  </action>
  <action name="actionPhysics">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="VoxCad.qrc">
     <normaloff>:/Icons/Icons/Sandbox.png</normaloff>:/Icons/Icons/Sandbox.png</iconset>
   </property>
   <property name="text">
    <string>Physics Sandbox</string>
   </property>
  </action>
  <action name="action3D_Brush">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="VoxCad.qrc">
     <normaloff>:/Icons/Icons/3DBrush.png</normaloff>:/Icons/Icons/3DBrush.png</iconset>
   </property>
   <property name="text">
    <string>3D Brush</string>
   </property>
  </action>
  <action name="actionTensile">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="VoxCad.qrc">
     <normaloff>:/Icons/Icons/Tensile.png</normaloff>:/Icons/Icons/Tensile.png</iconset>
   </property>
   <property name="text">
    <string>Tensile Test</string>
   </property>
  </action>
  <action name="actionVoxelCoordinates">
   <property name="text">
    <string>Voxel Coordinates</string>
   </property>
  </action>
  <action name="actionPaint_Bucket">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="VoxCad.qrc">
     <normaloff>:/Icons/Icons/Bucket.png</normaloff>:/Icons/Icons/Bucket.png</iconset>
   </property>
   <property name="text">
    <string>Paint Bucket</string>
   </property>
  </action>
  <action name="actionView_Tiled">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>View Tiled</string>
   </property>
  </action>
  <action name="actionGraphics_Enabled">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Graphics Enabled</string>
   </property>
  </action>
  <action name="actionKV6">
   <property name="text">
    <string>KV6</string>
   </property>
  </action>
  <action name="actionBenchmark">
   <property name="text">
    <string>Benchmark</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionShow_Triad">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Triad</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="VoxCad.qrc"/>
 </resources>
 <connections/>
</ui>
